web-dashboards/README.md

```diff
@@ -79,6 +84,41 @@ To start both services:
 
 `docker-compose -f docker-compose.yml up kibiter`
 
+### Kibiter 6
+
+Since the first review of the project, Kibiter 6 is used to build the web dashboards.
```

Comment:
Kibiter 6 is used to build --> has been used

Reply:
I prefer to use "is used to build" because now it is alive and I find it more clear that using "is used to" that "has been used". But maybe I am wrong. I won't change it in this review, and we can talk about it offline.